.github/ISSUE_TEMPLATE/05_mqissue.yml

```diff
@@ -0,0 +1,13 @@
+name: MQ Improvements Tracker
```

Comment:
If we don't want customers using this... we probably should consider whether we want to add this.

Reply:
<Had an offline conversation with @jsquire> We will add this new template and  use the wait and watch policy for testing.

Reply:
@jsquire let me know if you have any concerns.

Reply:
:shipit:

Reply:
Awesome, please approve :)